agentops/__init__.py

```diff
@@ -16,7 +16,7 @@ def init(api_key: Optional[str] = None,
          max_wait_time: Optional[int] = None,
          max_queue_size: Optional[int] = None,
          tags: Optional[List[str]] = None,
-         override=True,
+         override_llm_calls=True,
```

Comment:
Can we actually rename this to instrument_llm_calls? and make that change everywhere? I know this is outside of the scope here, but I think it would be a good time to make that change. 

Also note that this is actually a breaking change for users if they are currently setting this! Maybe add a optional here called override and put a deprecation notice as a logging statement and we can take it out in a few releases?

Reply:
True